Loop and Learn Volunteer edits

tmhastings opened the pull request:
Edits identified and suggested by Tina Hammer (@Stampertina), Meera Venkatesh (@MeeraVenkatesh), and Ellen Broadman (@ellenbroadman) from Loop and Learn in collaboration with Theresa Hastings (@tmhastings). While edits to the code were done by @tmhastings, the review, identification, and changes suggested were from the Loop and Learn team.

Edits from Loop and Learn team address:
- clarity
- grammar
- orphaned/incorrect links
- syntax
- formatting

Additional edits by @tmhastings 
- Add references to Loop and Learn's HCP Recommendation page and Integrated Diabetes Services alongside the already existing BC Diabetes reference in citations.md (commits [5c82f77](https://github.com/nightscout/trio-docs/pull/52/commits/5c82f77ff0148ed48e38c988b468fc4e27e8c74d), [db7fdc4](https://github.com/nightscout/trio-docs/pull/52/commits/db7fdc422efe24bfd8ebf3a60ca44c9d2109c75f), and [abf0901](https://github.com/nightscout/trio-docs/pull/52/commits/abf0901848d88a265526bcdc616610dd6fe809a1))
- Update CGM documentation to include Dexcom ONE+ compatibility from Issue #41  (commit [ee58798](https://github.com/nightscout/trio-docs/pull/52/commits/ee5879856be823d2b546a3c43bab9fe9f298247b))

Thank you to the amazing Loop and Learn team for your assistance!

tmhastings committed:
Update iphone.md

Correct formatting for admonitions

tmhastings committed:
Update pump.md

Correct link

tmhastings committed:
Update mainsettings.md

formatting correction

tmhastings committed:
Update trio.md

formatting corrections

tmhastings committed:
Update autosens-dynamic.md

change ICR to CR

tmhastings committed:
Update cgm.md

- update incorrect link
- grammar corrections

tmhastings committed:
Update Devices.md

- fix broken links
- grammar edits

tmhastings committed:
Update transition-qa.md

Clarify text

tmhastings committed:
Update transition-qa.md

- link correction
- grammar

tmhastings committed:
Update transition-qa.md

Add missing facebook link

tmhastings committed:
Update autosens-dynamic.md

correct link

tmhastings committed:
Update autotune.md

- reference "profile settings" earlier in document for clarity.
- minor grammar corrections

tmhastings committed:
Update Overview.md

Correct links by adding "../"

tmhastings committed:
Update Overview.md

Remove Dropdown menus to allow for functioning links for CR, ISF, and Basal.

tmhastings committed:
Update Overview.md

Formatting correction

tmhastings committed:
Update Overview.md

Formatting correction

tmhastings committed:
Update Overview.md

Still fixing the same formatting issue

tmhastings committed:
Merge pull request #1 from nightscout/dev

Sync with Current Dev

tmhastings committed:
Merge branch 'LnL-edits' into tmhastings-LnL-Edits

tmhastings committed:
Merge pull request #2 from nightscout/tmhastings-LnL-Edits

Sync with Loop and Learn edits branch from nightscout

tmhastings committed:
Update Devices.md

Correct links

tmhastings committed:
Update Devices.md

Correct phrasing

tmhastings committed:
Update iphone.md

Capitalize "devices" in title

tmhastings committed:
Update iphone.md

- Capitalize Headings
- Update Formatting

tmhastings committed:
Update transition-qa.md

- Corrected formatting
- Update Facebook Link

tmhastings committed:
Transitioning from iAPS

- Add "Transitioning from iAPS" to the list of AID systems
- Link corrections
- Headings formatting

tmhastings committed:
Update transition-qa.md

Correct formatting for links

tmhastings committed:
Update transition-qa.md

Remove malfunctioning links

tmhastings committed:
Update transition-qa.md

Fix "Dynamic Settings" Header

tmhastings committed:
Update transition-qa.md

Convert Dynamic Settings warning under iAPS 2.3.3 heading to a caution admonition format

tmhastings committed:
Update transition-qa.md

Change "carb" to "carbohydrate"

tmhastings committed:
Update transition-qa.md

add line break before "Coming from iOS Loop" section

tmhastings committed:
Update transition-qa.md

formatting

tmhastings committed:
Update transition-qa.md

Loop section format correction

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update transition-qa.md

Update Introduction

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update transition-qa.md

remove duplicate "chapter"

tmhastings committed:
Update autotune.md

tmhastings committed:
Update index.md

Move Autotune from Basic Settings to Advanced Settings

tmhastings committed:
Update Configure.md

Match reference to new menu name

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update Overview.md

tmhastings committed:
Update Overview.md

tmhastings committed:
Update cgm.md

tmhastings committed:
Update iphone.md

tmhastings committed:
Update citations.md

- Update "Community Support" section
- Add "Provider Support" section
  - Add Link to Loop and Learn's Supportive Provider List
  - Add Link to Integrated Diabetes Services
  - Move BC Diabetes under Provider Support

tmhastings committed:
Update citations.md

Add Bullets

tmhastings committed:
Update citations.md

Support Section updated to include Loop and Learn provider list and IDS where BC Diabetes is already mentioned.

tmhastings committed:
Update carbratios.md

Clarify recommended adjustments and replace recommended percentages with directional guidance.

tmhastings committed:
Update insulinsensitivities.md

Remove specific adjustment recommendations and replace with directional guidance.

tmhastings committed:
Update nightscout.md

Clarify text pulled from OpenAPS documentation.

tmhastings committed:
Update autosens-dynamic.md

Correct Typo

tmhastings committed:
Update build.md

grammar correction

tmhastings committed:
Update trio.md

correct typo

tmhastings committed:
Update dynamicsettings.md

grammar correction

tmhastings committed:
Update fatprotein.md

remove link

tmhastings committed:
Update autosens-dynamic.md

correct grammar

tmhastings committed:
Update pumpsettings.md

clarify Max Basal Setting

tmhastings committed:
Update carbratios.md

grammar correction

tmhastings committed:
Update build.md

Grammar correction

tmhastings committed:
Update build.md

clarifying edits

tmhastings committed:
Delete docs/EN/resources/img/compare.png

Remove the Comparison Table. It is out of date, but cannot be edited.

tmhastings committed:
Update troubleshoot.md

fix link

tmhastings committed:
Update dynamicsettings.md

Replace ICR with CR for consistency

tmhastings committed:
Update troubleshoot.md

link correction

tmhastings committed:
Update troubleshoot.md

correct link

tmhastings committed:
Update insulinsensitivities.md

correct markdown formatting

tmhastings committed:
Update trio.md

markdown correction

tmhastings committed:
Update mainsettings.md

tmhastings committed:
Update mainsettings.md

replace ICR with CR for consistency

tmhastings committed:
Update mainsettings.md

clarify Max IOB setting recommendation

tmhastings committed:
Update autosens-dynamic.md

tmhastings committed:
Update mainsettings.md

Add math syntax

tmhastings committed:
Update autosens-dynamic.md

replace ICR with CR for consistency

tmhastings committed:
Update dynamicsettings.md

tmhastings committed:
Update cgm.md

add Dexcom ONE+

tmhastings committed:
Merge branch 'dev' into LnL-edits

tmhastings committed:
Update index.md

tmhastings committed:
Update index.md

tmhastings committed:
Update New-User-Setup.md

Needed a closing statement.

marionbarker commented on docs/EN/Configuration/transition-qa.md:
```diff
@@ -108,53 +108,101 @@ You must read extensively and adapt to a new approach to diabetes management.
+<details>
+  <summary><b>Import Settings With Nightscout</b></summary>
+ 1. Connect Nightscout to your Trio app (If you've already connected your Nightscout site, skip to step 2)
+    * Open the 'Settings' Menu in the Trio app
```
Nightscout settings can only be imported if the display units in Trio and Nightscout agree.
If they don't match, you will get an error message when you try to impor.t

Not sure how you want to phrase this - default for new app is mmol/L, so this only affects folks using mg/dL. Settings, Preferences, Glucose units.

marionbarker commented on docs/EN/operate/build.md:
```diff
@@ -17,8 +17,8 @@ The Trio repository contains instructions for building the Trio app using a brow
 * `Identifier Names` will be: `FreeAPS`, `FreeAPSWatch`, `FreeAPSWatch WatchKit Extension`, `LiveActivityExtension`
-    * These names are the same as that used by iAPS, they are distinguised by the Identifier itself, which include the new BundleID for your Trio app
-    * If you build with Xcode before, they will start with XC and use the Identifier string rather than the Name
+    * These names are distinguished by the Identifier itself and include the new BundleID for your Trio app
```
Please do not modify build.md because this file is modified extensively by #47.
The modifications in this file will conflict with 47.

marionbarker commented on docs/EN/settings/devices/cgm.md:
```diff
@@ -13,25 +15,25 @@ CGM selection includes the following:
 
 ## xDrip (xDrip4iOS)
-To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../../operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
+To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
```
You need to go up 2 folders, then down one, i.e.

`(../../operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)`

marionbarker commented on docs/EN/settings/devices/cgm.md:
```diff
@@ -13,25 +15,25 @@ CGM selection includes the following:
 
 ## Glucose Simulator
 This option should only be used when testing the app and not using it to manage a live person (or animal).
 
```
I would add the user has no control over the glucose pattern provided by the simulator.

tmhastings replied:
I ended up rewriting the whole section on Glucose Simulator.

marionbarker commented on docs/EN/settings/devices/pump.md:
```diff
@@ -1,4 +1,4 @@
 # Pump
-This section allows you to select your insulin delivery system. For more information on supported pumps, see the following: [link](../../intro/requirements/pump.md). Older Omnipod and Medtronic devices require a Rileylink to translate communication between your phone's bluetooth signals and the pump's radiocommunication.
+This section allows you to select your insulin delivery system. For more information on supported pumps, see the following: [link](Getting-Started/pump.md). Older Omnipod and Medtronic devices require a Rileylink to translate communication between your phone's Bluetooth signals and the pump's radiocommunication.
```
Link should be `(../../Getting-Started/pump.md)`

marionbarker commented on docs/EN/operate/troubleshoot.md:
```diff
@@ -42,7 +42,7 @@ If you experience a roller coaster pattern of highs and lows after treatment, yo
 :::
 
-To avoid a second low, you can instruct Trio not to give any SMBs for a while after a low, giving your BG time to stabilize. Set up a [Temporary Target](./temptarget.md) preset that is a bit higher than your usual target (e.g., 7,5 mmol/L | 136 mg/dl). Ensure that [Allow SMB With High Temptarget](../settings/configuration/preferences/smbsettings.md) is disabled. Make the temporary target last for ...
+To avoid a second low, you can instruct Trio not to give any SMBs for a while after a low, giving your BG time to stabilize. Set up a [Temporary Target](../temptarget.md) preset that is a bit higher than your usual target (e.g., 7,5 mmol/L | 136 mg/dl). Ensure that [Allow SMB With High Temptarget](../settings/configuration/preferences/smbsettings.md) is disabled. Make the temporary target last fo ...
```
Link should be `(temptarget.md)` - the file is in the operate folder, same as troubleshoot.md

tmhastings committed:
Update build.md

Revert Loop and Learn team edits to allow merging with Nightscout/trio-docs PR #47

tmhastings committed:
Update transition-qa.md

Add step to verify glucose units for nightscout import

tmhastings committed:
Update cgm.md

fix link

tmhastings committed:
Update pump.md

fix link

tmhastings committed:
Update troubleshoot.md

fix link

tmhastings committed:
Update cgm.md

Redefine "Glucose Simulator" for clarification on what is does and does not do.

tmhastings committed:
Add files via upload

Add images of screen menu items for in-line images

tmhastings committed:
Delete docs/EN/Setup/img/Backfill Glucose.png

Uploaded to wrong img folder

tmhastings committed:
Delete docs/EN/Setup/img/Connect to Nightscout.png

Uploaded to wrong img folder

tmhastings committed:
Delete docs/EN/Setup/img/Connect.png

Uploaded to wrong img folder

tmhastings committed:
Delete docs/EN/Setup/img/Glucose Units.png

Uploaded to wrong img folder

tmhastings committed:
Delete docs/EN/Setup/img/Import Settings.png

Uploaded to wrong img folder

tmhastings committed:
Delete docs/EN/Setup/img/Nightscout Menu.png

Uploaded to wrong img folder

tmhastings committed:
Delete docs/EN/Setup/img/Preferences.png

Uploaded to wrong img folder

tmhastings committed:
Create img folder for Configuration

tmhastings committed:
Add files via upload

Add menu images for in-line use

marionbarker commented on docs/EN/settings/devices/cgm.md:
```diff
@@ -15,7 +15,7 @@ CGM selection includes the following:
 
 ## xDrip (xDrip4iOS)
-To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
+To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
```
My apologies - I did not notice the `html`. That needs to be `md`.

The correct link is `(../../operate/build.md#xdrip4ios-or-glucose-direct-as-cgm-source)`

tmhastings committed:
Delete docs/EN/Configuration/img directory

Failed attempt to add inline images. Will attempt at a later date.

marionbarker reviewed (changes requested):
Review each of the new commits. Found just one link error - all other links worked as expected.

tmhastings committed:
Update cgm.md

fix link
